lexicons/app/bsky/feed/defs.json

```diff
@@ -0,0 +1,83 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.defs",
+  "defs": {
+    "postView": {
+      "type": "object",
+      "required": ["uri", "cid", "author", "record", "replyCount", "repostCount", "upvoteCount", "downvoteCount", "indexedAt", "viewer"],
+      "properties": {
+        "uri": {"type": "string"},
+        "cid": {"type": "string"},
+        "author": {"type": "ref", "ref": "app.bsky.actor.defs#withInfo"},
+        "record": {"type": "unknown"},
+        "embed": {
+          "type": "union",
+          "refs": [
+            "app.bsky.embed.images#presented",
+            "app.bsky.embed.external#presented",
+            "app.bsky.embed.record#presented"
+          ]
+        },
+        "replyCount": {"type": "integer"},
+        "repostCount": {"type": "integer"},
+        "upvoteCount": {"type": "integer"},
+        "downvoteCount": {"type": "integer"},
+        "indexedAt": {"type": "string", "format": "datetime"},
+        "viewer": {"type": "ref", "ref": "#viewerState"}
+      }
+    },
+    "viewerState": {
+      "type": "object",
+      "properties": {
+        "repost": {"type": "string"},
+        "upvote": {"type": "string"},
+        "downvote": {"type": "string"}
+      }
+    },
+    "feedViewPost": {
```

Comment:
Wonder if now may be a good moment to sync-up variants we see around naming views, e.g. `#view`, `#viewBasic`, `#feedViewPost`, `#postView`.  Actually would be totally happy to punt on that or peel off a cleanup task.

Reply:
I'm down to do it now 👍 

I know that those are all different & it feels kinda ridiculous, but I actually think they may be right
- viewBasic: this is a view - modified by the word basic
- postView: this is a post - presented in a view
- feedViewPost: this is a constituent of the feed view - a post

I down to change feedViewPost & I actually tried to do that, but nothing felt quite right

Reply:
Ah yeah, I buy that!  And there will probably end-up being fewer `#view` with the new defs convention.  For example, what was once `post#view` is probably now `defs#postView`, and what as once `post#viewBasic` now `defs#postViewBasic`.